modules/ossm-upgrade-25-26-changes.adoc

```diff
@@ -0,0 +1,37 @@
+// Module included in the following assemblies:
+// * service_mesh/v2x/upgrading-ossm.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="ossm-upgrade-25-26-changes_{context}"]
+= Upgrade changes from version 2.5 to version 2.6
+
+//Jaeger disabled by default goes in "Upgrading"
+== {JaegerName} default setting change
+
+This release disables {JaegerName} by default for new instances of the `ServiceMeshControlPlane` resource.
+
+When updating existing instances of the `ServiceMeshControlPlane` resource to {SMProductName} version 2.6, {JaegerShortName} remains enabled by default.
+
+{SMProductName} 2.6 is the last release that includes support for {JaegerName} and {es-op}. Both {JaegerShortNamee} and {es-op} will be removed in the next release. If you are currently using {JaegerShortName} and {es-op}, you need to switch to {TempoName} and {OTELName}.
+
+== Envoy sidecar container default setting change
+
+To enhance pod startup times, Istio now includes a `startupProbe` in sidecar containers by default. The pod's readiness probes will not start until the Envoy sidecar has started.
+
+//Removed pending verification that SMCP supports disabling startupProbe.
+//You can disable the startup probe:
+
+//[source,yaml]
+//----
+//readinessInitialDelaySeconds: 1
```

Comment:
I asked about this in Slack before, and never got a response, but I'm not sure where this configuration would be set in the SMCP? Do we support these configuration settings or would they need to be set via meshConfig (and thus fall under the techPreview field)?